utils/config.go

```diff
@@ -0,0 +1,139 @@
+package utils
+
+import (
+	"fmt"
+	serialize "github.com/TRON-US/go-btfs-config/serialize"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/mitchellh/go-homedir"
+	_ "github.com/mitchellh/go-homedir"
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/tron-us/go-btfs-common/crypto"
+	"github.com/tron-us/go-common/v2/env"
+	"os"
+	"path/filepath"
+)
+
+type ApiConfigStruct struct {
+	PrivateKey string
+	PeerId     string
+	PublicKey  string
+}
+
+const (
+	// DefaultPathRoot is the path to the default config dir location.
+	DefaultApiPathRoot = "~/"
+	// DefaultConfigFile is the filename of the configuration file
+	DefaultApiConfigFile = ".config"
+	// EnvDir is the environment variable used to change the path root.
+	EnvDir = "BTFS_PATH"
+)
+
+var ApiConfig ApiConfigStruct
+
+// This function requires that the variables of this module be checked
+// for zero value before using them.
+func init() {
+	spath, err := apiConfigFilename()
+	if err != nil {
+		log.Error(err)
```

Comment:
need to panic or return, otherwise it continues execution

Reply:
sorry ramnant code. I should have removed.